pandas/tests/plotting/test_converter.py

```diff
@@ -71,27 +59,12 @@ def test_register_by_default(self):
         call = [sys.executable, "-c", code]
         assert subprocess.check_call(call) == 0
 
-    def test_warns(self):
-        plt = pytest.importorskip("matplotlib.pyplot")
```

Comment:
is there a way to check that we *have not* registered things?

Reply:
Hmm, we do that indirectly with the code check... If we haven't imported matplotlib, then we know we haven't registered anything.